### ISSUE TITLE
Eth1Monitor spams log messages during sync

### ISSUE THREAD

**Issue by jannikluhn** [ethereum/trinity#1506]
### What is wrong?

When starting `trinity-beacon` I get a seemingly endless number of log messages from the eth1 monitor (one per block):

```
Eth1Monitor  Eth1 Monitor got new eth1 block: Eth1Block(block_hash=b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x14^', number=5214, timestamp=193630), number of logs contained in the block: 1
```

I guess this is in part because of the fake provider we're currently using, but even with a normal provider we'll get a lot of blocks during sync at startup.

Link to code: https://github.com/ethereum/trinity/blob/680ef3737d8c78dd479542d80ad000487edf9b70/trinity/components/eth2/eth1_monitor/eth1_monitor.py#L145-L149

### How can it be fixed

Don't log one message per block, but instead one per successful fetch attempt.

**Comment by ralexstokes** [comment]
yeah i'd think we want to just batch messages from this component -- much like we would expect to periodically see peer counts written to the logs; we could periodically see how much progress happened w/ the eth1 monitor
